### DISCUSSION TITLE
Suitable CUDNN version for ort 1.14?

### DISCUSSION THREAD

**Discussion by oovm** [microsoft/onnxruntime#15037] [Other Q&A]
I query the CUDA/CUDNN/TensorRT version corresponding to ORT from here.

- [CUDA](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [TensorRT](https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#requirements)

It looks like the CUDNN version for `1.14/main` is missing, which version should I download?

Update the documentation if possible.

**Comment by pranavsharma** [comment]
https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements
